src/main/java/seedu/address/model/AddressBook.java

```
@@ -66,7 +66,7 @@ public ObservableList<Wedding> getWeddingList() {
     * @param persons the UniquePersonList to set
     */
     public void setPersons(UniquePersonList persons) {
-        this.currentWedding.setPersons(persons);
```

can cause null pointer error as current wedding can be null, im quite sure there are multiple functions that use setPersons, thoughts?

---

Oh I forgot to change the currentWedding references with getCurrentWedding(), my bad

---

I think cuz I'm doing getCurrentWedding(), it will throw a NoCurrentWeddingError instead of null pointer error

Actually this setPersons() isn't used so I might as well just remove it

---

okie can remove